carsus/io/cmfgen/hdfgen.py

```diff
@@ -0,0 +1,35 @@
+import os
+import glob
+from carsus import logger
+
+
+def h5dump(cmfgen_dir, patterns, parser, chunk_size=10):
+    files = []
+    for case in patterns:
+        path = cmfgen_dir + '/**/*{}*'.format(case)
+        files = files + glob.glob(path.replace('//', '/'), recursive=True)
+        files = [f for f in files if not f.endswith('.h5')]
+
+    n = chunk_size
+    files_chunked = [files[i:i+n] for i in range(0, len(files), n)]
+
+    # Divide read/dump in chunks for less I/O
+    for chunk in files_chunked:
+
+        _ = []
+        for fname in chunk:
+            try:
+                obj = parser.__class__(fname)
+                logger.info('Parsed {}'.format(fname.replace(cmfgen_dir + 'atomic/', '')))
+                _.append(obj)
+
+            except:
```

Comment:
well what errors do you expect?

Reply:
I think my problem is how to deal with errors I don't expect. Most of situations are contemplated in try/except blocks in `base.py`.

We have two 'kind' of parsers: 

- **energy levels, oscillator and collisional strengths:** Pandas-based. In the end we expect some Pandas error.

- **photoionization cross section:** handcrafted with Python standard library, and stored in Pandas dataframes. 


Then there are the exceptions related to `to_hdf` method.